cookietemple/create/templates/common_files/{{cookiecutter.commonName}}/docs/conf.py

```diff
@@ -1,50 +1,46 @@
 #!/usr/bin/env python
```

Comment:
why all the changes from `"` to `'` ;)?

Reply:
black for this template

the template uses common files

Reply:
I think you explained this already ^^